R/derive_params_growth_height.R

```diff
@@ -93,14 +92,13 @@
 #' advs <- dm_peds %>%
```

Comment:
can we add comments to this example please? as its quite a lot for users to follow with so much pre-processing before we even get to the example function call